lldb-eval/parser.cc

```diff
@@ -561,6 +531,19 @@ static std::unique_ptr<BuiltinFunctionDef> GetBuiltinFunctionDef(
     return std::make_unique<BuiltinFunctionDef>(identifier, return_type,
                                                 std::move(arguments));
   }
+  if (identifier == "__findnonnull") {
+    lldb::SBType return_type = ctx->GetBasicType(lldb::eBasicTypeInt);
+    std::vector<lldb::SBType> arguments = {
+        // The first argument should actually be "T**", but we don't support
```

Comment:
In `eval.cc` it says `T*`. If I understand correctly, it is supposed to accept any pointer type, but the single increment will always be the size of a pointer, instead of `sizeof(T)`?

Seems a bit weird to me, could we add a comment somewhere explaining this?

Reply:
That's a typo, fixed the comment and added descriptions for the functions.